tests/integration/devfile/cmd_delete_test.go

```diff
@@ -97,10 +97,12 @@ ComponentSettings:
 						// wait until the resources are deleted from the first delete
 						Eventually(string(commonVar.CliRunner.Run(getDeployArgs...).Out.Contents()), 60, 3).ShouldNot(ContainSubstring(deploymentName))
 						Eventually(string(commonVar.CliRunner.Run(getSVCArgs...).Out.Contents()), 60, 3).ShouldNot(ContainSubstring(serviceName))
-						stdOut = helper.Cmd("odo", "delete", "component", "--name", cmpName, "--namespace", commonVar.Project, "-f").ShouldPass().Out()
 					})
 					It("should output that there are no resources to be deleted", func() {
-						Expect(stdOut).To(ContainSubstring("No resource found for component %q in namespace %q", cmpName, commonVar.Project))
+						Eventually(func() string {
+							stdOut = helper.Cmd("odo", "delete", "component", "--name", cmpName, "--namespace", commonVar.Project, "-f").ShouldPass().Out()
+							return stdOut
+						}, 60, 3).Should(ContainSubstring("No resource found for component %q in namespace %q", cmpName, commonVar.Project))
```

Comment:
I think instead of this, you can wait for Pods to not be present.

Reply:
No, I would really want to test that the `delete component` command displays that no resources found

Reply:
I meant, you wait for the pods to not be present, and then run the delete command to ensure it gives the right message. Why do you choose otherwise?